### PR TITLE
New testing

### PR DESCRIPTION
**Pull request by camrossi**
New testing architecture

### CODE REVIEW

File: testing/selenium/selenium_ui_testing_dev_ndfc_v4.py [hunk 1]
```diff
@@ -234,22 +231,19 @@ def previous_pages(driver):
         'NDFC Login',
     ]
     for page in pages:
+        print(page)
```
Comment (samiib) [inline testing/selenium/selenium_ui_testing_dev_ndfc_v4.py:234]: Do you print to be here?

**Review by samiib — APPROVED** [review]
All LGTM :) Just had one comment.

**Comment by samiib** [comment]
Do you want to squash them all into one commit instead 70 at one go. 😆

File: testing/selenium/selenium_ui_testing_dev_ndfc_v4.py (hunk 1 above)
Comment (camrossi) [reply testing/selenium/selenium_ui_testing_dev_ndfc_v4.py:234]: Ah no was for debug ! Let me fix that

**Pull request merged**